packages/front-end/components/FactTables/FactTableModal.tsx

```diff
@@ -70,6 +78,24 @@ export default function FactTableModal({ existing, close }: Props) {
   useEffect(() => {
     if (!selectedDataSource || existing) return;
 
+    if (!selectedDataSource.projects || !selectedDataSource.projects.length) {
+      const filteredProjects = projects.filter((project) => {
+        // return permissionsUtil.canViewCreateFactTableModal(project.id);
+        return permissions.check("manageFactTables", project.id);
```

Comment:
Currently using the old permission check to avoid lint errors - if this branch gets pushed first, we can use the old style, otherwise I can upgrade to the new style once [this PR](https://github.com/growthbook/growthbook/pull/2321) is merged.